canonical-websites/www.ubuntu.com
# add context to forms

## Done

* it looks bigger than it is, sensible defaults are everywhere and will help with GA tracking
* added ‘?product=<name>’ to every contact-us form
* update server, container contact us to be more  contextual
* moved maas to use server/contact-us, not /cloud anymore

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: 
  - [/about/contact-us](http://0.0.0.0:8001/about/contact-us)
  - [/cloud/openstack](http://0.0.0.0:8001/cloud/openstack)
  - [/cloud/partners](http://0.0.0.0:8001/cloud/partners)
  - [/cloud/plans-and-pricing](http://0.0.0.0:8001/cloud/plans-and-pricing)
  - [/cloud/storage](http://0.0.0.0:8001/cloud/storage)
  - [/cloud/training](http://0.0.0.0:8001/cloud/training)
  - [/containers/contact-us](http://0.0.0.0:8001/containers/contact-us)
  - [/containers/docker-ubuntu](http://0.0.0.0:8001/containers/docker-ubuntu)
  - [/containers/](http://0.0.0.0:8001/containers/)
  - [/containers/kubernetes](http://0.0.0.0:8001/containers/kubernetes)
  - [/containers/lxd](http://0.0.0.0:8001/containers/lxd)
  - [/core](http://0.0.0.0:8001/core)
  - [/desktop/education](http://0.0.0.0:8001/desktop/education)
  - [/desktop/enterprise](http://0.0.0.0:8001/desktop/enterprise)
  - [/desktop/government](http://0.0.0.0:8001/desktop/government)
  - [/desktop/partners](http://0.0.0.0:8001/desktop/partners)
  - [/download/cloud/autopilot](http://0.0.0.0:8001/download/cloud/autopilot)
  - [/download/cloud/](http://0.0.0.0:8001/download/cloud/)
  - [/download/server/arm](http://0.0.0.0:8001/download/server/arm)
  - [/internet-of-things](http://0.0.0.0:8001/internet-of-things)
  - [/search](http://0.0.0.0:8001/search)
  - [/server/contact-us](http://0.0.0.0:8001/server/contact-us)
  - [/server/hyperscale](http://0.0.0.0:8001/server/hyperscale)
  - [/server/](http://0.0.0.0:8001/server/)
  - [/server/maas](http://0.0.0.0:8001/server/maas)
  - [/server/management](http://0.0.0.0:8001/server/management)
  - [/support/contact-us](http://0.0.0.0:8001/support/contact-us)
  - [/support/esm](http://0.0.0.0:8001/support/esm)
  - [/support](http://0.0.0.0:8001/support)
  - [/support/plans-and-pricing](http://0.0.0.0:8001/support/plans-and-pricing)
- click on any contact us page and see that it goes to one.
